src/pages/userManagement/components/EditCollaborationDialog.jsx

```diff
@@ -0,0 +1,224 @@
+  summaryStates,
+} from '../../../models/collaboration/model';
+
+const coreStateChoices = [
+  {
+    value: 'view',
+    labelId: 'COLLABORATION_STATE_VIEW',
+  },
+  {
+    value: 'edit',
+    labelId: 'COLLABORATION_STATE_EDIT',
+  },
+];
+
+const fullStateChoices = [
+  ...coreStateChoices,
+  {
+    value: 'revoked',
+    labelId: 'COLLABORATION_STATE_REVOKED',
+  },
+];
+
+function FormattedUserName({ name }) {
+  return name || <FormattedMessage id="UNNAMED_USER" />;
+}
+
+function getPatchCollaborationOperations(
+  newSummaryState,
+  collaboration,
+) {
+  switch (newSummaryState) {
+    case summaryStates.revoked:
+      return [
+        {
+          op: 'replace',
+          path: '/managed_view_permission',
+          value: {
+            permission: collaborationStates.revoked,
+          },
+        },
+      ];
+    case summaryStates.edit:
+      return [
+        {
+          op: 'replace',
+          path: '/managed_edit_permission',
+          value: {
+            permission: collaborationStates.approved,
+          },
+        },
+      ];
+    case summaryStates.view: {
+      const operations = [
+        {
+          op: 'replace',
+          path: '/managed_view_permission',
+          value: {
+            permission: collaborationStates.approved,
+          },
+        },
+      ];
+
+      if (isEditApproved(collaboration)) {
+        operations.push({
+          op: 'replace',
+          path: '/managed_edit_permission',
+          value: {
+            permission: collaborationStates.revoked,
+          },
+        });
+      }
+
+      return operations;
+    }
+    default:
+      return [];
+  }
+}
+
+export default function EditCollaborationDialog({
+  collaboration,
+  open,
+  onClose,
+}) {
+  const [collaborationState, setCollaborationState] = useState(
+    getSummaryState(collaboration),
+  );
+  const members = Object.values(collaboration?.members || {});
+
+  const {
+    mutate: patchCollaboration,
+    isLoading,
+    error,
+    clearError,
+  } = usePatchCollaboration();
+
+  useEffect(() => {
+    setCollaborationState(getSummaryState(collaboration));
+  }, [collaboration]);
+
+  const stateChoices =
+    isViewApproved(collaboration) || isViewRevoked(collaboration)
+      ? fullStateChoices
+      : coreStateChoices;
+
+  function handleClose() {
+    // cleanup
+    setCollaborationState('');
+    clearError();
+
+    // close
+    onClose();
+  }
+
+  return (
+    <StandardDialog
+      maxWidth="md"
+      titleId="EDIT_COLLABORATION"
+      open={open}
+      onClose={handleClose}
+    >
+      <DialogContent>
+        <dl style={{ display: 'flex', flexWrap: 'wrap', gap: 24 }}>
+          {members.map((member, index) => {
+            const style = { flexGrow: 1 };
+            // The maxWidth aligns the user information with selection input
+            if (index === 1) style.maxWidth = 280;
+
+            return (
+              <div key={member?.guid} style={style}>
```

Comment:
`style={{ flexGrow: 1, maxWidth: index === 1 ? 280 : 'unset' }}`
 (if you like it better, not a huge deal either way)

Reply:
Thanks. I changed it in https://github.com/WildMeOrg/codex-frontend/pull/451/commits/0e0918d3e8d4c5901242ff18c8f4edecf9b494b3.